src/Team-Services-Bot.Api.UnitTests/Dialogs/RootDialogTests.cs

```diff
@@ -9,15 +9,6 @@
 
 namespace Vsar.TSBot.UnitTests
```

Comment:
Why no tests for rootdialog? It seems like rootdialog has lots of functionality.